docs/config/main.css

```diff
@@ -24,41 +24,39 @@
 
 body {
    	margin: 5px 10px 10px 5px;
-	background-color: ligthgrey; 
+	background-color: lightgray;

```

Comment:
im pretty sure it makes no difference but could we keep consistent on using 'grey' instead of 'gray'